examples/managed-node-groups/main.tf

```diff
@@ -0,0 +1,468 @@
+#------------------------------------------------------------------------
```

Comment:
This file probably needs formatting.

Reply:
terraform fmt updated. Comments formatted as well